hmrc/telemetry-git-hooks
# Add check-tool-versions as a pre-commit git hook

What did we do?
--

1. Created a telemetry git hooks repo so that we don't need to copy paste our git hooks into every repo!

Evidence of work
--

1. Tested passing & failing with alert-config
<img width="450" alt="image" src="https://user-images.githubusercontent.com/18111914/229073726-89e4e6e2-9617-4fc5-8052-0963ba4f475c.png">

Next Steps
--

1. Update all repos using check-tool-versions to use this repo
2. Copy over all of our other local git hooks
3. Add PR check and deployer for git repo tagging
4. Update documentation

Risks
--

1. none

Collaboration
--

Co-authored-by: Think Stack Limited <29373851+thinkstack@users.noreply.github.com>
